tools/mason/MasonTest.chpl

```diff
@@ -527,7 +527,7 @@ proc testFile(file, ref result, show: bool) throws {
         erroredTestNames: list(string),
         testsPassed: list(string),
         skippedTestNames: list(string);
-    var localesCountMap: map(int, int, parSafe=true);
+    var localesCountMap: map(int, int, parSafe=false);
```

Comment:
@ben-albrecht Is this adjustment to localesCountMap OK? As far as I can tell the map is never used in a parallel context.